packages/xrpl/src/models/transactions/transaction.ts

```diff
@@ -212,7 +211,7 @@ export function validate(transaction: Record<string, unknown>): void {
 
   if (
     !isEqual(
-      decode(encode(tx)),
+      tx,
```

Comment:
I think by remove the `decode(encode(tx))` block, Mayukha was referring to rmoving lines 212 -219 - @mvadari is that correct? (I think that makes sense to me as well if that's what you meant - we check this anyway when we submit, and the error isn't very verbose here)
```
if (
   !isEqual(
...
    throw new ValidationError(`Invalid Transaction: ${tx.TransactionType}`)
  }
```

Reply:
@JST5000 the entire block as we noted above you mean?
Also, for failing checks - somehow I can not reproduce the same thing in the local environment (win10)

Reply:
> I think by remove the `decode(encode(tx))` block, Mayukha was referring to removing lines 212 -219 - @mvadari is that correct? (I think that makes sense to me as well if that's what you meant - we check this anyway when we submit, and the error isn't very verbose here)

Yes, that is what I meant.

Reply:
@mvadari sorted it and linter + unit test failing due to decode/encode